src/transform/activation.jl

```diff
@@ -0,0 +1,132 @@
+
+# -------------------------------------------------------------
+# Repurposed from https://github.com/tbreloff/OnlineAI.jl
+# -------------------------------------------------------------
+
+abstract Activation <: Transformation
+
+value(activation::Activation, input::AbstractVector) = Float64[value(activation, Σ) for Σ in input]
```

Comment:
This will force everything to `Float64`which takes the users ability away to use `Float32`. I don't think you need to specify the array type of `value` were typestable. So `[value(activation, Σ) for Σ in input]``should be fine.

it works for `sin` after all

```julia
julia> [sin(x) for x in 1:5]
5-element Array{Float64,1}:
  0.841471
  0.909297
  0.14112 
 -0.756802
 -0.958924
```